content/python/concepts/built-in-functions/terms/print/print.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'print()'
+Description: 'Takes in data such as a string or an int converted to string using str() and prints it to the standard output device.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The print() function takes in a string or any other element such as int converted to a string using str() and displays those values to the standard output device.
+
+## Syntax
+
+```py
+print("Hello, World!")
```

Comment:
```suggestion
print(object)
```

I recommend using a placeholder here. Take a look at the info under the "Syntax" heading for [w3schools](https://www.w3schools.com/python/ref_func_print.asp) or some other (built-in function)[https://www.codecademy.com/resources/docs/python/built-in-functions] Dos entries.

We can create another heading for the examples and include your line 22 as an ex. as well:

## Example 1

Short description:

```py
Code here
```

## Example 2

...and so on